SmartDeviceLink/public/SDLVideoStreamingCapability.h

```diff
@@ -30,7 +31,22 @@ NS_ASSUME_NONNULL_BEGIN
  @param scale The scaling factor the app should use to change the size of the projecting view
  @return A SDLVideoStreamingCapability object
  */
-- (instancetype)initWithPreferredResolution:(nullable SDLImageResolution *)preferredResolution maxBitrate:(int32_t)maxBitrate supportedFormats:(nullable NSArray<SDLVideoStreamingFormat *> *)supportedFormats hapticDataSupported:(BOOL)hapticDataSupported diagonalScreenSize:(float)diagonalScreenSize pixelPerInch:(float)pixelPerInch scale:(float)scale;
+- (instancetype)initWithPreferredResolution:(nullable SDLImageResolution *)preferredResolution maxBitrate:(int32_t)maxBitrate supportedFormats:(nullable NSArray<SDLVideoStreamingFormat *> *)supportedFormats hapticDataSupported:(BOOL)hapticDataSupported diagonalScreenSize:(float)diagonalScreenSize pixelPerInch:(float)pixelPerInch scale:(float)scale __deprecated_msg("Use initWithPreferredResolution:maxBitrate:supportedFormats:hapticDataSupported:diagonalScreenSize:pixelPerInch:scale:preferredFPS instead");
+
+/**
+ Convenience init for creating a video streaming capability with all parameters.
+
+ @param preferredResolution The preferred resolution of a video stream for decoding and rendering on HMI
+ @param maxBitrate The maximum bitrate of video stream that is supported, in kbps
+ @param supportedFormats Detailed information on each format supported by this system, in its preferred order
+ @param hapticDataSupported True if the system can utilize the haptic spatial data from the source being streamed
+ @param diagonalScreenSize The diagonal screen size in inches
+ @param pixelPerInch The diagonal resolution in pixels divided by the diagonal screen size in inches
+ @param scale The scaling factor the app should use to change the size of the projecting view
+ @param preferredFPS The preferred frame rate per second of the head unit
+ @return A SDLVideoStreamingCapability object
+ */
+- (instancetype)initWithPreferredResolution:(nullable SDLImageResolution *)preferredResolution maxBitrate:(int32_t)maxBitrate supportedFormats:(nullable NSArray<SDLVideoStreamingFormat *> *)supportedFormats hapticDataSupported:(BOOL)hapticDataSupported diagonalScreenSize:(float)diagonalScreenSize pixelPerInch:(float)pixelPerInch scale:(float)scale preferredFPS:(uint32_t)preferredFPS;
```

Comment:
We are trying to use the inits generated by the RPC Spec generator script. That init allows us to pass a `nil` value for `preferredFPS`.

Reply:
Understood. I have updated SDLVideoStreamingCapability.h based on the generated code (updated preferredFPS relevant code only).